test/basic.html

```diff
@@ -80,6 +96,15 @@
           done();
         }, 1);
       });
+
+      test('can be styled with different sizes', function() {
+        var r2 = fixture('WithDifferentSizes');
+        var normal = r2[0].getBoundingClientRect();
+        var giant = r2[1].getBoundingClientRect();
+        var tiny = r2[2].getBoundingClientRect();
+        assert.isTrue(5 <= tiny.height < normal.height < giant.height <= 50);
```

Comment:
JS doesn't have chained comparison operator sugar, so these comparisons will need to be split up. [Also, you might want to update your copy of paper-checkbox.](https://github.com/PolymerElements/paper-checkbox/pull/94) :)